src/index.js

```diff
@@ -1,22 +1,18 @@
 import './style.css';
-import addScore from './addScore';
+import postData, { showMessage } from './addScore';
 
 let scores = [];
 
-if (localStorage.scores) {
-  scores = JSON.parse(localStorage.getItem('scores'));
-}
-
 const scoresContainer = document.querySelector('.scores');
 
-const showScores = (scores) => {
+const showScores = async () => {
   scoresContainer.innerHTML = '';
   scores.forEach((each) => {
     const li = document.createElement('li');
     li.className = 'item';
     const nameSpan = document.createElement('span');
     nameSpan.className = 'player-name';
-    nameSpan.textContent = each.name;
+    nameSpan.textContent = each.user;
 
     const scoreSpan = document.createElement('span');
     scoreSpan.className = 'player-score';
```

Comment:
- [ ] OPTIONAL: While your app works perfectly, I don't think you need to make this `showScores` an `async` function as it is not using `await` for any promise. You may check your app if it works without the `async` here. :pray:

Reply:
Oh! Thank you very much @henatan99:pray:
I added await before but forgot to remove the async when I removed the await keyword:smile: